src/main/java/ladder/RandomLineGenerator.java

```diff
@@ -0,0 +1,53 @@
+package ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomLineGenerator implements LineGenerator {
+
+    private static final Random RANDOM = new Random();
+
+    private List<Boolean> points;
+
+    @Override
+    public List<Boolean> run(int ladderSize) {
+        points = init(ladderSize);
+        generateRandomHorizonLine();
+        return points;
+    }
+
+    private List<Boolean> init(int ladderSize) {
+        List<Boolean> booleans = new ArrayList<>(ladderSize);
+        for (int i = 0; i < ladderSize; i++) {
+            booleans.add(i, false);
+        }
+        return booleans;
+    }
+
+    private void generateRandomHorizonLine() {
+        int pointSize = points.size();
+
+        for (int index = 0; index < pointSize; index++) {
+            addHorizonLine(index);
+        }
+    }
+
+    private void addHorizonLine(int index) {
+        if (this.verifyAddHorizon(index)) {
```

Comment:
아주 소소한 의견이긴 합니다만 `this`는 제거 되도 될 것 같습니다 😄 
```suggestion
        if (verifyAddHorizon(index)) {
```

그리고 검증하기 보다는 실제 `method`가 하는 역할이 `horizon`을 `add`할 수 있는지를 체크하는 `method`이므로 
좀 더 네이밍이 직관적이게 바뀌면 좋을 것 같습니다. (예를들어 `isHorizonAddable`같은 이름으로)
네이밍은 항상 어려운 것 같습니다 😅